include/fmt/chrono.h

```diff
@@ -1389,16 +1389,54 @@ inline std::chrono::duration<Rep, std::milli> get_milliseconds(
 #endif
 }
 
-template <typename Rep, typename Period,
-          FMT_ENABLE_IF(std::is_floating_point<Rep>::value)>
-inline std::chrono::duration<Rep, std::milli> get_milliseconds(
-    std::chrono::duration<Rep, Period> d) {
-  using common_type = typename std::common_type<Rep, std::intmax_t>::type;
-  auto ms = mod(d.count() * static_cast<common_type>(Period::num) /
-                    static_cast<common_type>(Period::den) * 1000,
-                1000);
-  return std::chrono::duration<Rep, std::milli>(static_cast<Rep>(ms));
-}
+template <class Duration> class subsecond_helper {
+  /// Returns the amount of digits according to the c++ 20 spec
+  /// In the range [0, 18], if more than 18 fractional digits are required,
+  /// then we return 6 for microseconds precision
```

Comment:
Please replace `///` with `//` since this is an internal API.

Reply:
```suggestion
  // Returns the amount of digits according to the c++ 20 spec
  // In the range [0, 18], if more than 18 fractional digits are required,
  // then we return 6 for microseconds precision
```